src/example-types/metricGroups/pivot.js

```diff
@@ -109,22 +109,21 @@ let buildQuery = async (node, schema, getStats) => {
     let sortField = getSortField(sort)
 
     return _.reduce(
-      async (children, group) => {
-        // Calculating subtotal metrics at each group level if not flattening or drilling down
-        // Support for per group stats could also be added here - merge on another stats agg blob to children based on group.stats/statsField or group.values
-        if (!node.flatten && !node.drilldown)
```

Comment:
Could we get a comment here in the PR as to why were can get rid of the flattening?  Just for posterity?

Reply:
After discussing, flatten is no longer a thing in contexture.